CONTRIBUTING.rst

```diff
@@ -22,6 +22,11 @@ You can now clone your fork of scVelo and install the development mode
     git clone https://github.com/YOUR-USER-NAME/scvelo.git
     cd scvelo
     git checkout --track origin/develop
+
+In general, the installation can be completed with
+
+.. code:: bash
+
```

Comment:
Let's have two sections/paragraphs here, instead. One for Unix (Linux and macOS) and one for windows. This way it'll be easier for people to see the info relevant to them.